CGPs/cgp-0076.md

```diff
@@ -0,0 +1,155 @@
+---
+cgp: 76
+title: LatAm-W3-DAO-Proposal
+date-created: 2023-03-25
+author: '@juancamp1987 in behalf of Mila Rioja'
+status: DRAFT
+discussions-to: https://forum.celo.org/t/latam-w3-dao-proposal/4494
+governance-proposal-id: TBD
+date-executed: TBD
+---
+<!---
+YAML ends
+-->
+
+<!---
+Beginnning of the proposal
+-->
+# Abstract
+To spread awareness about the Celo protocol and increase its adoption, grow the Celo community with initiatives in the most used languages in LatAm (Portuguese and Spanish! 3), and help nurture the next cohort of great initiatives on Web3.
+Celo has one of the most global communities in crypto. 
+In order to further expand decentralization and the localization of the protocol in different regions and cultures, we would love to empower local communities to lead local growth.
+We believe that folks on the ground are the most equipped to do this and that a grassroots approach will benefit the growth of Celo and its community.
+
+
+# Proposal
+
+## Goal
+Bring to life next gen of dApps on Celo, and contribute with the community in projects towards raising awareness about the Celo protocol and increase its adoption, grow the Celo community with initiatives in the most used languages in LatAm (Portuguese and Spanish!),
+
+Celo has one of the most global communities in crypto. In order to further expand decentralization and the localization of the protocol in different regions and cultures, we must empower local communities to lead local growth. Decision power and initiatives must be shifted to folks on the ground who are the most equipped to collab and achieve a grassroots approach that will benefit the growth of Celo and its community.
+
+## Amount Requested
+$500k worth of CELO
+
+## Areas of Focus
+
+### Accelerating Growth: Web3 Venture Builder (Solution Design + Project Development and Shipping + Capital)**
+Create a Web3 Venture Studio to launch the next gen of Web3 initiatives by: (i) helping expressive Web2 companies launch on Celo and (ii) nurturing native web3 initiatives.
+
+#### Budget Ask: $280,000
+- Technical Support (encompasses tech development and business support)
+- Technical Support: 300 hours / Average professional hour at $80
+- Extras (e.g. back office/operations) $40,000
+
+#### Success Metrics:
+- 8 Web3 Dev Studio Participants across the region
+- 100% Project Conversion Rate to Celo
+- Transaction volume >$500M after 1 year
+
+
+### Collabs for Awareness, Education and Hackathons
+Help onboard new community members and developers, and guide them through the ecosystem, by collaborating with ecosystem founders and partners, doing Hackathons, meetups and other learning initiatives.
+
+#### Budget Ask: $120,000
+We are looking to accomplish at least 12 initiatives around the next year (6 in portuguese and 6 in spanish). Ecosystem partners must submit their proposals for such and the total budget will be allocated to them. We will co-organize the initiative together and report to the community.
```

Comment:
We are looking to accomplish at least 12 initiatives around the next year (6 in Portuguese and 6 in Spanish). Ecosystem partners (i.e. Founders on Celo) must submit their proposals for such and budget will be allocated after open calls for discussion. We may co-organize the initiatives together and report to the community.